jobs/ScheduleBackupJob.js

```diff
@@ -82,6 +82,7 @@ class ScheduleBackupJob extends BaseJob {
     const options = _.omit(job.attrs.data, 'trigger', 'type');
     return backupStore
       .listBackupsOlderThan(options, config.backup.retention_period_in_days)
```

Comment:
I understood that we are now keeping base backups for retention_period + 1 days. How is it handled? 
Also, currently the transaction logs uploaded by service are not deleted as part of it, will that be part of another PR?

Reply:
this PR would be updated for transaction log deletion.

Reply:
Transaction log deletion?

Reply:
so should we not merge this PR then?